ci/scripts/r_valgrind.sh

```diff
@@ -44,3 +49,5 @@ fi
 # https://github.com/microsoft/LightGBM/blob/fa6d356555f9ef888acf5f5e259dca958ca24f6d/.ci/test_r_package_valgrind.sh#L20-L85
 
 popd
+
+popd
```

Comment:
Is this a typo or is it needed again here?

Reply:
Are popds even needed here? My assumption is shell scripts don't influence the state after completing.

Reply:
I added another `pushd` up above, so I added this down here to match the flow — nut you're right that this shouldn't change the state or anything so is probably not needed.